RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -451,9 +451,11 @@ __RestorationOptimization __calculate_objective_values(int hp_goal, int mp_goal,
 	float hp_restored_per_use()
 	{
 		float restored_amount = metadata.hp_restored;
-		if(metadata.restores_variable_hp == __RESTORE_ALL)
+		// treat cocoon as a full HP restore since rarely > 1000 max HP in run
+		if((metadata.restores_variable_hp == __RESTORE_ALL) || (metadata.name == "cannelloni cocoon"))
```

Comment:
if you want to adjust cocoon to be considered a restore all, it should be done in the datafile.

which incidentally has no compile functionality at the moment. Something I will be working on soon